blocklist-client/src/config/default.toml

```diff
@@ -0,0 +1,7 @@
+[server]
+host = "127.0.0.1"
+port = 3030
+
+[risk_analysis]
+api_url = "http://example.com/api"
+api_key = "api-key-here"
```

Comment:
Should we provide defaults for these? I'd rather have the app fail on startup if these aren't set instead of starting it with misconfigured API parameters.

Reply:
That makes sense, I put them for a reference/easier placeholder for running it till I add the README but I'll add the README and remove those dummy defaults